File: projects/AlbumFotos/App_Code/Foto.vb
Imports Microsoft.VisualBasic

Public Class Foto

    Private _fotoID As Integer
    Public Property FotoID() As Integer
        Get
            Return _fotoID
        End Get
        Set(ByVal value As Integer)
            _fotoID = value
        End Set
    End Property
    Private _titulo As String
    Public Property Titulo() As String
        Get
            Return _titulo
        End Get
        Set(ByVal value As String)
            _titulo = value
        End Set
    End Property
    Private _descricao As String
    Public Property Descricao() As String
        Get
            Return _descricao
        End Get
        Set(ByVal value As String)
            _descricao = value
        End Set
    End Property
    Private _fotoDados As Byte()
    Public Property FotoDados() As Byte()
        Get
            Return _fotoDados
        End Get
        Set(ByVal value As Byte())
            _fotoDados = value
        End Set
    End Property
End Class

File: projects/BlobASPNet/BlobASPNet/App_Code/Foto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

public class Foto
{
    private int _fotoID;
    private string _titulo;
    private string _descricao;
    private byte[] _fotoDados;

    public int FotoID
    {
        get { return _fotoID; }
        set { _fotoID = value; }
    }

    public string Titulo
    {
        get { return _titulo; }
        set { _titulo = value; }
    }

    public string Descricao
    {
        get { return _descricao; }
        set { _descricao = value; }
    }

    public byte[] FotoDados 
    {
        get { return _fotoDados; }
        set { _fotoDados = value; }
    }
}